src/serviceFacades/tools.js

```diff
@@ -158,6 +160,49 @@ function deleteTools({ ids: toolIds }) {
     );
 }
 
+function getAdminToolRequests(_, { order, orderBy, page, rowsPerPage }) {
+    // Determine if the request is supposed to be ordered.
+    const isOrdered = order && orderBy;
+
+    // Determine if the request is supposed to be paginated.
+    const isPaginated = (page || page === 0) && rowsPerPage;
+
+    // Build the object containing the query parameters.
+    const params = {};
+    if (isOrdered) {
+        params["sort-dir"] = order.toUpperCase();
+        params["sort-field"] = orderBy.toLowerCase();
+    }
+    if (isPaginated) {
+        params["limit"] = rowsPerPage;
+        params["offset"] = page * rowsPerPage;
+    }
+
+    return callApi({
+        endpoint: `/api/admin/tool-requests`,
+        method: "GET",
+        params: params,
+    });
+}
```

Comment:
`GET` `/admin/tool-requests` supports pagination but right now it doesn't return the `total` field so it is not set up to use pagination for now.